docs/sources/tempo/traceql/query-editor/_index.md

```diff
@@ -3,7 +3,10 @@ title: Write TraceQL queries in Grafana
 menuTitle: Write TraceQL queries in Grafana
 description: Learn how to create TraceQL queries in Grafana using the query editor and search.
 aliases:
-  - /docs/tempo/latest/traceql/construct-query
+  - ./ # /docs/tempo/<TEMPO_VERSION>/traceql/query-editor
```

Comment:
```suggestion
```

Reply:
Thank you for these updates. I thought I had read in the toolkit that we are supposed to place aliases for the current page, which is why there are aliases for the current page location. However, I can't find where I read that...

Reply:
That's true but only as a safety for unversioned documentation in case someone forgets to add an alias after moving a file.

For versioned documentation `./` is always gonna point to the current page so won't provide a safety net redirect if the page moves because it won't point to the old place.